base/simdloop.jl

```diff
@@ -85,6 +85,36 @@ function compile(x, ivdep)
     end
 end
 
+"""
+    @simd
+
+Annotate a `for` loop to allow the compiler to take extra liberties to allow loop re-ordering
+
+!!!warning
+    This feature is experimental and could change or disappear in future versions of Julia.
+    Incorrect use of the `@simd` macro may cause unexpected results.
+
+The object iterated over in a `@simd for` loop should be a one-dimensional range or a CartesianIndices iterator.
+By using `@simd`, you are asserting several properties of the loop:
+
+    * It is safe to execute iterations in arbitrary or overlapping order, with special consideration for reduction variables.
```

Comment:
So is it safe to remove this bullet point now?

Reply:
No I think that is still a property of `@simd` w.r.t to floating-point operations